scanner/src/main/kotlin/scanners/BoyterLc.kt

```diff
@@ -154,13 +154,12 @@ class BoyterLc(name: String, config: ScannerConfiguration) : LocalScanner(name,
         }
     }
 
-    override fun getRawResult(resultsFile: File): JsonNode {
-        return if (resultsFile.isFile && resultsFile.length() > 0L) {
+    override fun getRawResult(resultsFile: File) =
```

Comment:
prefer explicit return type here

Reply:
For me, the return type is obvious here as the "else" branch has `EMPTY_JSON_NODE`.